mm2src/coins/eth.rs

```diff
@@ -745,6 +745,7 @@ impl SwapOps for EthCoin {
         fee_addr: &[u8],
         amount: &BigDecimal,
         min_block_number: u64,
+        _uuid: &[u8],
```

Comment:
I assume will be used for later usage? Or it's simply to respect the interface for `send_taker_fee`?

Reply:
As of now, it's simply to respect the interface, yes. But it will be useful in the future when we implement a more advanced swap protocol.